src/sdk/http.ts

```diff
@@ -58,7 +58,14 @@ export class Http {
         if (str.indexOf(')]}\'\n') === 0) {
           str = str.substring(5);
         }
-        error = JSON.parse(str) as ApiErrorJson;
+        try {
+          error = JSON.parse(str) as ApiErrorJson;
+        } catch(e) {
+          error = {
+            message: reason.response!!.status + " " + reason.response!!.statusText
```

Comment:
is this for a 404 case with your new endpoint?

Reply:
Yeah when company logo is not set it returns an error ans it was failing at this point.